CRM/Core/DAO/Phone.php

```diff
@@ -289,7 +289,7 @@ public static function &fields() {
         'phone_type_id' => [
           'name' => 'phone_type_id',
           'type' => CRM_Utils_Type::T_INT,
-          'title' => ts('Phone Type'),
+          'title' => ts('Phone Type ID'),
```

Comment:
Push-me pull-me again. I've been changing these fields to user-friendly names in the metadata (e.g. without the "ID"). That's best for form builder, search builder, and forms in general.